cmd/webhook/admission/testdata/rules.yaml

```diff
@@ -0,0 +1,15 @@
+rules:
+  - reject: Missing application label, see https://example.test/reference/labels-selectors/#application
+    when: |
+      request.kind.kind in ["Ingress", "RouteGroup"] && (
+        !has(request.object.metadata.labels) ||
+        !has(request.object.metadata.labels.application) ||
+        request.object.metadata.labels.application == ''
+      )
```

Comment:
We may also implement
```yaml
rules:
  - warning: Feature X will be deprecated
    when:  request.kind.kind == "RouteGroup" && ...
```
that would allow request with warnings, see https://github.com/zalando/skipper/commit/9281783bc5242325317cdbf0ada97ebf2a4136ee